USAGE.md

```diff
@@ -122,72 +122,104 @@ aws lambda invoke \
+    /dev/stdout
 
-- Example of single Dataset listing request
+{"status_code": 200, "body": [{"created_at": "2021-02-01T13:38:40.776333+0000", "id": "cb8a197e649211eb955843c1de66417d", "title": "Auckland_2020", "updated_at": "2021-02-01T13:39:36.556583+0000"}]}
+```
 
-  ```console
-  $ aws lambda invoke \
-      --function-name "${ENV}-datasets" \
-      --payload '{"http_method": "GET", "body": {"id": "cb8a197e649211eb955843c1de66417d"}}' \
-      /dev/stdout
+### Single dataset listing request
 
-  {"status_code": 200, "body": {"created_at": "2021-02-01T13:38:40.776333+0000", "id": "cb8a197e649211eb955843c1de66417d", "title": "Auckland 2020", "updated_at": "2021-02-01T13:39:36.556583+0000"}}
-  ```
+```console
+$ aws lambda invoke \
+    --function-name "${ENV}-datasets" \
+    --payload '{"http_method": "GET", "body": {"id": "cb8a197e649211eb955843c1de66417d"}}' \
+    /dev/stdout
 
-- Example of Dataset delete request
+{"status_code": 200, "body": {"created_at": "2021-02-01T13:38:40.776333+0000", "id": "cb8a197e649211eb955843c1de66417d", "title": "Auckland_2020", "updated_at": "2021-02-01T13:39:36.556583+0000"}}
+```
 
-  ```console
-  $ aws lambda invoke \
-      --function-name "${ENV}-datasets" \
-      --payload '{"http_method": "DELETE", "body": {"id": "cb8a197e649211eb955843c1de66417d"}}' \
-      /dev/stdout
+### Dataset delete request
 
-  {"status_code": 204, "body": {}}
-  ```
+```console
+$ aws lambda invoke \
+    --function-name "${ENV}-datasets" \
+    --payload '{"http_method": "DELETE", "body": {"id": "cb8a197e649211eb955843c1de66417d"}}' \
+    /dev/stdout
+
+{"status_code": 204, "body": {}}
+```
+
+### Changing the dataset title by creating a copy of the latest dataset version
+
+This is the simplest way to change a dataset title:
+
+1. [Create a new dataset](#Dataset-creation-request)
+1. [Create a new dataset version](#Dataset-Version-creation-request) for the dataset created above,
+   using a `metadata_url` pointing to the latest version of the original dataset.
+1. [Wait for the import to finish](#Import-Status-Endpoint-Usage-Examples).
+1. Optional: if the original dataset can be removed at this point (or sometime in the future),
+   please let the Geostore product team know, and we'll arrange it.
+
+### Changing the dataset title by renaming and moving the files
+
+This process is more cumbersome and time-consuming than the above, and results links to the old
+dataset being broken rather than slowly phased out, but can be followed if necessary.
+
+1. Send a request to the product team asking for a rename, specifying the current and new title of
+   the dataset. The product team then takes care of the rest of the process:
+   1. Schedule necessary downtime and notify users.
+   1. Turn off external access to the whole or part of the system to avoid any conflicts.
+   1. Run a manual rename of all the files in the relevant dataset.
+   1. Use the `${ENV}-datasets` endpoint to rename the dataset in the database.
```

Comment:
Does 'renaming' a dataset exist or work on the datasets endpoint yet? There's nothing further down in the docs about it?

Reply:
The functionality is on the "datasets" endpoint, but we've left it out of the usage documentation since it's not meant to be used by end users. TBC whether we want to just remove it from public view, since I guess it'd be a fair bit of work to make that particular functionality accessible only to admins.

Reply:
Sounds good to me.